WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginUtils.java

```diff
@@ -28,7 +29,8 @@ public static boolean isPluginFeatureAvailable(SiteModel site) {
         return site.isWPCom()
                && SiteUtils.hasNonJetpackBusinessPlan(site)
                && site.getHasCapabilityManageOptions() // Automated Transfer require admin capabilities
-               && !site.getUrl().contains(".wordpress.com") // Automated Transfers require custom domains
+               // Automated Transfers require custom domains
```

Comment:
Since we modified this check, can we improve on this comment? Something like the following might work:

```suggestion
               // Automated Transfers require custom domains, so if the user has a `xyz.wordpress.com` site, custom domain registrations feature needs to be enabled
```